docker/token/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM registry.cern.ch/cmsweb/cmsweb:20220201-stable
```

Comment:
WE must replace this image since it is TOO big:
```
registry.cern.ch/cmsweb/cmsweb      20220201-stable    77a74bf8539e   2 weeks ago     2.57GB
```
and we only need shell, curl and kubectl tools to get the token. I suggest to replace cmsweb with debian, something like that (I didn't test it):
```
FROM debian:stable-slim
RUN apt-get update && apt-get -y install curl

# Install latest kubectl for using with crons
RUN curl -k -O -L https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
RUN mv kubectl /usr/bin
RUN chmod +x /usr/bin/kubectl
```
This will give image with O(100MB) image size instead of 2.7GB.